### DISCUSSION TITLE
What data should you use to create nowcasts for this hub?

### DISCUSSION THREAD

**Discussion by nickreich** [reichlab/variant-nowcast-hub#99] [General]
After chatting with a few prospective modeling teams, I wanted to clarify what data we are anticipating teams will use to generate their nowcasts for this hub. 

After some discussion, we have decided not to host the datasets needed in the hub repo itself because then we’re in the business of data curation, and people will start to rely on our pipelines in ways that we are not prepared to support. There already are some good pipelines for these data, and we don't want to create duplicative data stores.

That said, here are the options, current as of Monday, October 7, 2024:
 - we are working on a python utility to create clade counts by state based on the latest NextStrain data. This discussion will be updated with details about this when it is ready for public use, hopefully sometime in the next week.
 - In the interim, [NextStrain makes datasets](https://github.com/seattleflu/forecasts-ncov-states/blob/main/ingest/README.md#outputs) that are largely the same as the count files that our utility will produce. In the table at that page (see screenshot below), you would want to use the file in “open (GenBank) > Nextstrain clades > USA”.

![image (1)](https://github.com/user-attachments/assets/0fd574bd-6ef7-4aeb-8eec-2b0918703a59)

We note that the NextStrain files above may use slightly different methods for filtering which sequences are aggregated and creating the counts, so it is possible that these data may not be exactly the same as the ones that our pipeline will eventually create for evaluation, but we expect differences (if any) to be minimal.

**Comment by nlinton** [comment]
Have you been tracking what data source teams use at all?

**Reply by nickreich** [reply]
Not in any way other than what teams have written in the required methods "write-up" that is part of the model metadata.
